components/financial_assistance/app/views/financial_assistance/applications/preferences.html.erb

```diff
@@ -190,7 +190,7 @@
             <div class="col-xs-12">
               <div class="back-link">
                 <i class="fa fa-caret-left fa-2x" aria-hidden="true"></i>
-                <a href="javascript:void(0);" onclick="notifyUserPrompt(this)" data-path="<%= financial_assistance.edit_application_path(@application) %>"> <b>BACK TO ALL HOUSEHOLD MEMBERS</b> </a> <span>(Go back and work on another person)</span>
+                <a href="javascript:void(0);" class="back-to-household-link" data-path="<%= financial_assistance.edit_application_path(@application) %>"> <b>BACK TO ALL HOUSEHOLD MEMBERS</b> </a> <span>(Go back and work on another person)</span>
```

Comment:
replace javascript:void(0) with #. Check for `event.preventDefault()` at the top of the function.